app/assets/stylesheets/content/editor/_ckeditor.sass

```diff
@@ -32,6 +32,13 @@ ckeditor-augmented-textarea .op-ckeditor--wrapper
   figure.image
     margin: 1em 0
 
+  figure.table
+    // Ensure we break apart words in table cells that
+    // are restricted in width (OP#33524)
+    td[style*=";width:"],
+    td[style^="width:"]
```

Comment:
As discussed, I checked whether we need this specific rule definition. The answer is yes, because `break-all` breaks words instead of moving them completely to a new line (even if there would be enough space in the next line). This would then look like a bug

Reply:
Thanks for checking :+1: